templates/vue/cypress/support/utils.js

```diff
@@ -0,0 +1,48 @@
+// TODO: Add node generator
+// TODO: Add link generator
+
+export const SITE_URL = "http://localhost:8888"
```

Comment:
This might be different on other machines - please make it customizable somehow - maybe through an environment file that you would add to .gitignore? And then include instructions to add it in the README file.